plugins/parsers/csv/parser.go

```diff
@@ -84,7 +84,7 @@ func (record metadataPattern) Less(i, j int) bool {
 func (p *Parser) initializeMetadataSeparators() error {
 	// initialize metadata
 	p.metadataTags = map[string]string{}
-	p.metadataSeparatorList = []string{}
+	p.metadataSeparatorList = make([]string, 0)
```

Comment:
Why do we need this? The list is initialized _again_ in line 97 and not used before, isn't it?

Reply:
Removed.